README.tr.md

```
@@ -0,0 +1,485 @@
+node gemini-eval.mjs --file ./jds/my-job.txt
+node agent-inbox.mjs add "..."   # bir sonraki oturum için isteği kuyruğa alır
+npm run gemini:eval -- "JD text here"
+```
+
+> **Ücretsiz katman:** Her iki seçenek de faturalandırma olmadan çalışır. Yerel CLI Google OAuth kullanır; API betiği `gemini-2.5-flash` kullanır (dakikada 15 istek, günde 1M token ücretsiz).
+
+## Kullanım
+
+career-ops paylaşılan bir komut yönlendiricisi kullanır. Slash komutlarını kaydeden CLI'lerde şöyle görünür:
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)